src/sentry/models/release.py

```diff
@@ -219,6 +225,7 @@ def filter_by_stage(
         value,
         project_ids: Sequence[int] = None,
         environments: List[str] = None,
+        negated: bool = False,
```

Comment:
Looks like this is unused. Do we actually need to pass this? Looks like we already handle negations in here.